legate/core/communicator.py

```diff
@@ -120,7 +120,16 @@ def __init__(self, runtime: Runtime) -> None:
         )
         self._init_cpucoll = library.LEGATE_CORE_INIT_CPUCOLL_TASK_ID
         self._finalize_cpucoll = library.LEGATE_CORE_FINALIZE_CPUCOLL_TASK_ID
-        self._tag = library.LEGATE_CPU_VARIANT
+        num_omps = int(
```

Comment:
let's move this to `Runtime` and make the communicator use it, as I think it's better to consolidate all tunable lookups in one place for code maintenance.

Reply:
I have pushes a new commit for it. Besides omps, I also add tunable lookups for CPUs and GPUs.